spec/controllers/hq/hq_controller_spec.rb

```diff
@@ -30,4 +30,16 @@ def foobar_action
       get :foobar_action
     end
   end
+
+  describe 'instantiates' do
+    specify 'navigation buttons' do
+      expect(HqController::NAVIGATION_BUTTONS).to be_present.and be_a_kind_of Array
+      expect(HqController::NAVIGATION_BUTTONS.first).to be_a_kind_of Hash
+      expect(HqController::NAVIGATION_BUTTONS.first[:text]).to be_present.and be_a_kind_of String
+      expect(HqController::NAVIGATION_BUTTONS.first[:href]).to be_present.and be_a_kind_of String
+      expect(HqController::NAVIGATION_BUTTONS.first[:path_regex]).to be_present.and be_a_kind_of Regexp
+      expect(HqController::NAVIGATION_BUTTONS.first[:glyph]).to be_present.and be_a_kind_of String
+    end
+  end
```

Comment:
Is there much value in this test? Especially considering that it only tests a single button